src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/versionMatcher/versionObjectAmzn.hpp

```diff
@@ -56,19 +56,19 @@ class VersionObjectAmzn final : public IVersionObject
     static bool match(const std::string& version, Amzn& output)
     {
         std::smatch parserMatches;
-        if ((std::regex_search(version, parserMatches, m_parserRegex) == false) || (parserMatches.size() != 5))
```

Comment:
Note: Demanding the presence of the 5 regex groups is not necessary in my opinion:
- The regex is already demanding the presence of one group. If the five groups are necessary, why not modifying the regex to demand all of them?
- A lot of versions of ALAS with which we are working won't match the 5 groups. We'd discard a lot of cases.

I've personally checked that all the ALAS versions pass this match method. Maybe in the future we could make this more restrictive.